### PR TITLE
Adopt octoprint systemcommandeditor

### PR DESCRIPTION
**Pull request by synman**

### CODE REVIEW

File: _plugins/systemcommandeditor.md [hunk 1]
```diff
@@ -4,14 +4,16 @@ layout: plugin
 id: systemcommandeditor
 title: System Command Editor
 description: This plugin makes System Commands editable through the OctoPrint WebUI
-author: Marc Hannappel (Salandora)
+authors:
+- Shell M Shrader (maintainer)
```
Comment (jneilliii) [inline _plugins/systemcommandeditor.md:8]: not sure you want to put your authorship this way, when filtering the plugin repo by author you will have two separate listings: https://plugins.octoprint.org/by_author/#shell-m-shrader

**Review by jneilliii — CHANGES_REQUESTED** [review]
See code comment...

File: _plugins/systemcommandeditor.md (hunk 1 above)
Comment (synman) [reply _plugins/systemcommandeditor.md]: I think that does it.

**Review by jneilliii — APPROVED** [review]

**Comment by synman** [comment]
tysir!

**Pull request merged**